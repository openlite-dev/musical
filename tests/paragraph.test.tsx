import * as React from 'react'
import renderer from 'react-test-renderer'

import UParagraph from '../src/components/paragraph'

it(`renders correctly`, () => {
  const tree = renderer.create(<UParagraph>Snapshot test!</UParagraph>).toJSON()

  expect(tree).toMatchSnapshot()
})
