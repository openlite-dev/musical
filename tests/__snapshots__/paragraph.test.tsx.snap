// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    [
      {
        "fontSize": 16,
        "letterSpacing": 0,
      },
    ]
  }
>
  Snapshot test!
</Text>
`;
